application/src/main/java/org/togetherjava/tjbot/features/mediaonly/MediaOnlyChannelListener.java

```diff
@@ -54,20 +56,30 @@ private boolean messageHasNoMediaAttached(Message message) {
                 && !message.getContentRaw().contains("http");
     }
 
-    private RestAction<Message> dmUser(Message message) {
+    private MessageCreateData createNotificationMessage(Message message) {
         String originalMessageContent = message.getContentRaw();
+
         MessageEmbed originalMessageEmbed =
                 new EmbedBuilder().setDescription(originalMessageContent)
                     .setColor(Color.ORANGE)
                     .build();
 
-        MessageCreateData dmMessage = new MessageCreateBuilder().setContent(
-                "Hey there, you posted a message without media (image, video, link) in a media-only channel. Please see the description of the channel for details and then repost with media attached, thanks 😀")
+        return new MessageCreateBuilder().setContent(message.getAuthor().getAsMention()
+                + " Hey there, you posted a message without media (image, video, link) in a media-only channel. Please see the description of the channel for details and then repost with media attached, thanks 😀")
```

Comment:
please update ur base branch correctly. ur PR contains unrelated changes from ur other PR.

ur supposed to create different independent branches to avoid such stuff

Reply:
means I did fork it I need to sync my fork?

Reply:
@AthithyaIthayan no. ur PR is from ur `develop` to our `develop`. instead, u should locally create a branch `bugfix/website_widget` based on our `develop` (which doesnt include ur other changes yet). and then have the PR from that branch to our develop.

that said, since u now have contributor-permission, u can actually also work on a branch directly on our repo instead of working with a fork.

Reply:
whatever u do, ensure that the PR only has changes for this feature and not the other